### DISCUSSION TITLE
Mobile Theme & Styling

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#7] [Ideas]
I would like this app to support the following:

View Ports:
  1920x1080 (Desktop)
  1884x905 (Desktop; medium screen; based on 1920x1080)
  1366x768 (Laptops)
  1024x768 (Tablets ?)
  375x677 (Mobile ; most cell phones and tablets including iPhone, Samsung )
  360x640 (Mobile ; other XS cell phones)

**Comment by 4-Leafs-Code** [comment]
Hey friend! I saw this project on http://github-help-wanted.com/  and I am also building a react social network: https://github.com/FearlessTech/Reentry feel free to check out our code.

**Reply by gbowne1** [reply]
@4-Leafs-Code Thats cool.

**Comment by gbowne1** [comment]
@balajik
